bayesianflows.tex

```diff
@@ -41,31 +41,29 @@
 
-Bayesian approaches provide a pathway to powerful models that can transform how we understand our systems as large-scale ecological data increasingly becomes available for more diverse systems and new questions. Ecologists recognize this potential and are increasingly using Bayesian approaches, but are often inadequately prepared to notice or manage the pitfalls of such models. Many of these pitfalls can be avoided by approaching analyses through specific workflows \citep{betanworkflow,grinsztajn2021,vandeschoot2021}, which themselves are built on a process of how to do not just statistics, but how to do science \citep{box1976science}. % Maybe address workflow comment here? 
-
-These approaches move away from a focus on null hypothesis testing, towards estimating effect sizes, using models calibrated (see Table \ref{tab:glossary}) and better understood through simulating data at multiple steps---using a number of skills often reserved in ecology more for `theorists' than empirical ecologists. Given the societal demands on ecologists today, we argue this divide is antiquated---especially as the average modern ecologist is computational, we believe they also need the skills to build and understand their own models. Breaking the training divide between theoretical and empirical ecologists, we argue, would yield greatly increased flexibility in what models can be fit and improved insights into the underlying systems.
+These approaches move away from a focus on null hypothesis testing, towards estimating effect sizes, using models calibrated (see Table \ref{tab:glossary}) and better understood through simulating data at multiple steps---
+% WDP: The bits from "using models calibrated" aren't clear to me - maybe can you rephrase?
+using a number of skills often reserved in ecology more for `theorists' than empirical ecologists.
+% WDP: Here I think we're missing that nice link with fundamental theory, which I think will bring the 'previous generation' of scientists along with us for the ride here
+Given the societal demands on ecologists today, we argue this divide is antiquated---especially as the average modern ecologist is computational, we believe they also need the skills to build and understand their own models. Breaking the training divide between theoretical and empirical ecologists, we argue, would yield greatly increased flexibility in what models can be fit and improved insights into the underlying systems.
```

Comment:
Lots of good comments on the intro, which I mostly integrated while shortening the intro also.